pkg/kubelet/dockertools/docker.go

```diff
@@ -325,7 +326,7 @@ func NewDockerContainerCommandRunner(client DockerInterface) ContainerCommandRun
 }
 
 func (p dockerPuller) Pull(image string) error {
-	image, tag := parseImageName(image)
+	_, tag := parsers.ParseRepositoryTag(image)
```

Comment:
I can provide a fix for this in a couple of hours, or if someone wants to do it sooner, feel free.